src/main/java/fr/xephi/authme/util/Utils.java

```diff
@@ -106,4 +107,15 @@ public static boolean isCollectionEmpty(Collection<?> coll) {
     public static boolean isEmailEmpty(String email) {
         return StringUtils.isEmpty(email) || "your@email.com".equalsIgnoreCase(email);
     }
+
+    /**
+     * Returns whether the given string as an UUID or null
+     *
+     * @param string the uuid to parse
+     * @return parsed UUID if succeed or null
+     */
+    public static UUID parseUuidSafely(String string) {
```

Comment:
Use UuidUtils directly

Reply:
I'd like to keep a separation between Optional method and nullable method, maybe you want me to move this method to UuidUtils instead?

Reply:
Yeah, that sounds better :)